src/main/kotlin/com/demonwav/mcdev/platform/forge/version/ForgeVersion.kt

```diff
@@ -10,8 +10,8 @@
 
 package com.demonwav.mcdev.platform.forge.version
 
-import com.demonwav.mcdev.util.gson
 import com.demonwav.mcdev.util.SemanticVersion
+import com.demonwav.mcdev.util.gson
```

Comment:
This seems weird to me. Does uppercase sort before lowercase? Seems like imports should be sorted case-insensitive to me.

Reply:
That's what IntelliJ IDEA does when sorting the imports. It seems like they are sorted case-sensitive, but since this is what your IntelliJ code style does, I'm not sure we should bother and keep this different manually